sw/device/tests/spi_device_sleep_test.c

```diff
@@ -155,9 +170,18 @@ bool test_main(void) {
       &pinmux, kTopEarlgreyMuxedPadsIoa7, kDifPinmuxPadKindMio));
 
   LOG_INFO("Setting SPI_DIO1 to low when sleeping");
+  CHECK_DIF_OK(dif_pinmux_pad_sleep_enable(
+      &pinmux, kTopEarlgreyDirectPadsSpiDeviceSd0, kDifPinmuxPadKindDio,
```

Comment:
Ci is failing due to this tests failing on the FPGA. To fix it you have to clear the state on this pin after wakeup just like line 195.

Reply:
Done [here](https://github.com/lowRISC/opentitan/blob/ea463e945361223b42daafc5415c2732ede5584c/sw/device/tests/spi_device_sleep_test.c#L193-L208)